package.json

```diff
@@ -141,10 +141,10 @@
     "lint-staged": "^10.3.0",
     "markdownlint-cli": "^0.23.2",
     "prettier": "*",
-    "react": "^16.13.1",
-    "react-dom": "^16.13.1",
+    "react": "^16.13.1 || ^17.0.0",
```

Comment:
We should probably use 17 going forward and upgrade the rest of the dev deps as well.

Reply:
I don't know what other extra dev-deps you want tu upgrade. Plz let me know.

Reply:
You left a note that:
> Styleguidist already launched updating react@16 to react@17

so I suppose we should update that as well (`react-styleguidist`)? These are just dev dependencies anyway, so it won't really matter to end users.

Reply:
There's also:
```
    "@testing-library/react": "^11.0.2",
    "@testing-library/react-hooks": "^3.4.1",
    ...

    "@types/react": "^16.9.49",
    "@types/react-dom": "^16.9.8",
```

Reply:
I mean, despite of upgrading react to v17, styleguideist is currently rendering and working fine.

Reply:
I mean, despite of upgrading react to v17, styleguideist is currently rendering and working fine.

Reply:
Sure. In any case, I suggest using `^17.0.0` (not `^16.13.1 || ^17.0.0`) and update the @types packages as well and the testing lib if necessary. I'll have a subsequent PR to update the rest of the deps.

Reply:
updating...

I'll send it on an our.